# ALL:ALL - Se agrega -lstdc++ y -lm a Libs en rubberband.pc.in para que funcione el enlace estatico cruzado sin librerias estandar faltantes.
--- librubberband/rubberband.pc.in
+++ librubberband/rubberband.pc.in
@@ -6,5 +6,5 @@
 Name: rubberband
 Version: %VERSION%
 Description: 
-Libs: -L${libdir} -lrubberband
+Libs: -L${libdir} -lrubberband -lstdc++ -lm
 Cflags: -I${includedir} 
